test/integration/models/linode/test_linode.py

```diff
@@ -315,11 +315,11 @@ def test_linode_boot(create_linode):
 def test_linode_resize(create_linode_for_long_running_tests):
     linode = create_linode_for_long_running_tests
 
-    wait_for_condition(10, 100, get_status, linode, "running")
+    wait_for_condition(10, 120, get_status, linode, "running")
```

Comment:
Any chance we could increase this to something like 240 just to err on the side of caution?